canvas_modules/common-canvas/__tests__/common-properties/controls/numberfield-test.js

```diff
@@ -260,14 +260,13 @@ describe("numberfield control works correctly", () => {
 		fireEvent.change(doubleNumber, { target: { value: "0.3" } });
 		expect(controller.getPropertyValue(numPropertyId)).to.equal(0.3);
 	});
-	// input has unexpected behavior
-	it.skip("should not allow a bad value to be set in a field", async() => {
+	it("should not allow a bad value to be set in a field", async() => {
```

Comment:
You can remove this test. This test is same as `"should allow a null value to be set in an integer field"`.